tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go

```diff
@@ -244,6 +247,8 @@ func (f *eventBusFlow) checkSubscriptionReady() error {
 }
 
 func (f *eventBusFlow) publishTestEvent() error {
+	randomInt = rand.Intn(100)
+	f.log.Debugf("Publish random int: %v", randomInt)
```

Comment:
It would have been better to 
* not use some kind of state variable instead to inject the value for pre and post upgrade as a function parameter.
* use clear strings such as `pre-upgrade-paylaod + rand.int()` and `post-upgrade + rand.int()` to have better debugging support.